src/client/app/components/visual-unit/CreateVisualUnitModalMapComponent.tsx

```diff
@@ -0,0 +1,219 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import * as React from 'react';
+import * as d3 from 'd3';
+import { useEffect } from 'react';
+import { useIntl } from 'react-intl';
+import { useAppSelector } from '../../redux/reduxHooks';
+import { selectAllUnits } from '../../redux/api/unitsApi';
+import { selectConversionsDetails } from '../../redux/api/conversionsApi';
+
+/**
+ * Visual graph component that shows the relationship between units and conversions
+ * entered by an admin. (Uses conversion data for conversion links).
+ * @returns D3 force graph visual
+ */
+export default function CreateVisualUnitMapModalComponent() {
```

Comment:
I'm inclined not to have "Map" in the component name since map has another meaning in other places. Maybe just remove it here and in CreateCikVisualMapComponent.